.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
I'm not a troll @OogieBoogieInJSON 
I have a simple question.
Sorry, if I offended you.